scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/UploadServlet.java

```diff
@@ -122,23 +142,15 @@ private void upload(HttpServletRequest request, HttpServletResponse response) {
                 fi.delete();
             }
 
-            LOGGER.warn("sessionId=" + sessionId);
-            LOGGER.warn("bucketName=" + bucketName);
-            LOGGER.warn("workflowName=" + workflowName);
-            LOGGER.warn("job=" + job);
-
             if (bucketName != null && workflowName != null) {
                 fetchFromCatalogAndWriteResponse(bucketName, workflowName, sessionId, response);
             } else {
                 writeResponse(job, response);
             }
 
         } catch (Exception e) {
-            try {
-                String msg = e.getMessage().replace("<", "&lt;").replace(">", "&gt;");
-                response.getWriter().write(msg);
-            } catch (IOException ignored) {
-            }
+            String msg = e.getMessage().replace("<", "&lt;").replace(">", "&gt;");
+            writeSilently(response, msg);
         } finally {
             if (job != null)
                 job.delete();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use "java.nio.Files#delete" here for better messages on error conditions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4042)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "delete". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS899)